Fujiwara edit

kfujiwara opened the pull request:
Most of the comments are updated.
I cannot decide about the following comments.

SECDIR review of draft-ietf-dnsop-avoid-fragmentation-16
|Section 1. Introduction, last paragraph, page 3: The first sentence is fine. I
|don't understand just what the rest of the paragraph is saying or why it is
|useful. A "path MTU" can be "obtained" (not set?) through "static
|configuration, server routing hints, ..."? Is this configuration/hint
|affecting transit devices so as to limit/expand the path MTU? What's going on

  I would like to change the last paragraph in the Introduction: remove the second and following sentences.

Murray Kucherawy's Discuss
| * Define "large / small" better.

I cannot define.

| "Protocol compliance considerations"

| * Would be nice to see reporting recommendations, perhaps that make
| the failure an internal cost for the failing component?... would not
| want a repeat of dmarc though.

I cannot rewrite

Robert Wilton's Discuss
|    R8.  DNS responses may be dropped by IP fragmentation.  Upon a
|    timeout, to avoid resolution failures, UDP requestors MAY retry using
|    TCP or UDP with a smaller EDNS requestor's maximum UDP payload size
|    per local policy.
|
| Again, I think that this document would be clearer if this was a SHOULD rather
| than a MAY.

Can we change the "MAY" as "SHOULD" ?

kfujiwara committed:
fujiwara's 2024/1/29-31 edit

kfujiwara committed:
minor fixes

kfujiwara committed:
fixed references:
  Normative reference: refered from main part
  Informative reference: refered from Appendix

vixie reviewed (changes requested):
the REMOVE for path MTU discovery is the only controversial part of this commit. we must leave open the possibility that some day in the future the old ethernet frame size limit (1500) will be considered too absurd, and the network will evolve beyond it. perhaps we can change "can be" to "might be"? otherwise this specification can be referenced to bolster the cast against such evolution.

kfujiwara committed:
Changed as vixie's comment:
  https://github.com/ietf-wg-dnsop/draft-ietf-dnsop-avoid-fragmentation/pull/39#pullrequestreview-1886975881

kfujiwara committed:
edited iesgcomment.txt to remove mail headers

kfujiwara committed:
R8: changed as "SHOULD" retry.
R9,10,12: added text to describe the upper limit.

kfujiwara commented:
Changed: One of Robert Wilton's Discuss: R8. UDP requestors MAY retry -> SHOULD
Added text: One of Murray Kucherawy's Discuss: Define "large / small" better.

vixie commented:
Thanks 


p vixie 


On Feb 26, 2024 02:48, Kazunori Fujiwara ***@***.***> wrote:

@kfujiwara requested your review on: #39 Fujiwara edit.

—
Reply to this email directly, view it on GitHub, or unsubscribe.
You are receiving this because your review was requested. Message ID: ***@***.***>

kfujiwara committed:
Renumbered recommendation numbers

kfujiwara committed:
Paul Vixie's review
